### PR TITLE
Update for Symphony 4.x

### PR DESCRIPTION
**Pull request by alexnantel88**
Fix installation error (Symphony Warning: Declaration of extension_remote_selectbox::update($previousVersion) should be compatible with Extension::update($previousVersion = false))

### CODE REVIEW

File: extension.meta.xml [hunk 1]
```diff
@@ -22,8 +22,12 @@
 		<!-- None -->
 	</dependencies>
 	<releases>
+		<release version="2.0.0" date="TBA" min="4.0.0" max="4.x.x" php-min="5.6.x" php-max="7.x.x">
+			- Update for Symphony 4.x
```
Comment (nitriques) [inline extension.meta.xml]: Please remove this line, we will release before v4

File: extension.meta.xml [hunk 2]
```diff
@@ -22,8 +22,12 @@
 		<!-- None -->
 	</dependencies>
 	<releases>
+		<release version="2.0.0" date="TBA" min="4.0.0" max="4.x.x" php-min="5.6.x" php-max="7.x.x">
```
Comment (nitriques) [inline extension.meta.xml]: set today's date, set the min to "2.3.3" and max to "2.x.x" Thanks !

File: extension.meta.xml (hunk 1 above)
Comment (alexnantel88) [reply extension.meta.xml]: Done.

File: extension.meta.xml (hunk 2 above)
Comment (alexnantel88) [reply extension.meta.xml]: Done.

File: extension.meta.xml [hunk 3]
```diff
@@ -22,8 +22,11 @@
 		<!-- None -->
 	</dependencies>
 	<releases>
+		<release version="1.1" date="2018-04-18" min="2.3.3" max="2.x.x">
```
Comment (nitriques) [inline extension.meta.xml]: We now use [semver](https://semver.org/) so the version must be `1.1.0` :P

Comment (alexnantel88) [reply extension.meta.xml]: Done.

**Pull request merged**